src/phases/egg-summary-phase.ts

```diff
@@ -0,0 +1,53 @@
+import BattleScene from "#app/battle-scene.js";
+import { Phase } from "#app/phase.js";
+import { Mode } from "#app/ui/ui.js";
+import EggHatchSceneHandler from "#app/ui/egg-hatch-scene-handler.js";
+import { EggHatchData } from "#app/data/egg-hatch-data.js";
```

Comment:
```suggestion
import BattleScene from "#app/battle-scene";
import { Phase } from "#app/phase";
import { Mode } from "#app/ui/ui";
import EggHatchSceneHandler from "#app/ui/egg-hatch-scene-handler";
import { EggHatchData } from "#app/data/egg-hatch-data";
```
NIT: Same comment as above